spec/system/visitor/manager_view_condo_full_visitors_list_spec.rb

```diff
@@ -0,0 +1,279 @@
+require 'rails_helper'
+
+describe 'Manager view condo full visitors list' do
+  it 'from visitors agenda' do
+    manager = create :manager
+    first_condo = create :condo, name: 'Condomínio Teste'
+    tower = create :tower, condo: first_condo, name: 'Torre A'
+    second_condo = create :condo
+    first_resident = create :resident, full_name: 'Alberto Silveira', residence: tower.floors[0].units[0]
+    second_resident = create :resident, full_name: 'Maria Silveira', residence: tower.floors[0].units[1]
+    first_visitor = create :visitor, condo: first_condo, resident: first_resident,
+                                     visit_date: Time.zone.today, full_name: 'João da Silva', identity_number: '12467'
+    second_visitor = create :visitor, condo: first_condo, resident: first_resident,
+                                      category: :employee, recurrence: :weekly,
+                                      visit_date: Time.zone.today, full_name: 'Maria Oliveira', identity_number: '45977'
+    third_visitor = create :visitor, condo: first_condo, resident: second_resident,
+                                     visit_date: 1.day.from_now, full_name: 'Marcos Lima', identity_number: '12345'
+    fourth_visitor = create :visitor, condo: second_condo,
+                                      visit_date: Time.zone.today, full_name: 'Juliana Ferreira'
+
+    login_as manager, scope: :manager
+    visit find_condo_visitors_path first_condo
+    click_on 'Ver Lista Completa'
+
+    expect(page).to have_content 'Visitantes/Funcionários cadastrados no Condomínio Teste'
+    expect(current_path).to eq all_condo_visitors_path first_condo
+    within("#visitor-#{first_visitor.id}") do
+      expect(page).to have_content 'João da Silva'
+      expect(page).to have_content '12467'
+      expect(page).to have_content 'Visitante'
+      expect(page).to have_content 'Torre A - 11'
+      expect(page).to have_content 'Alberto Silveira'
+      expect(page).to have_content I18n.l(Time.zone.today)
+    end
+    within("#visitor-#{second_visitor.id}") do
+      expect(page).to have_content 'Maria Oliveira'
+      expect(page).to have_content '45977'
+      expect(page).to have_content 'Funcionário'
+      expect(page).to have_content 'Torre A - 11'
+      expect(page).to have_content 'Alberto Silveira'
+      expect(page).to have_content I18n.l(Time.zone.today)
+      expect(page).to have_content 'Semanal'
+    end
+    within("#visitor-#{third_visitor.id}") do
+      expect(page).to have_content 'Marcos Lima'
+      expect(page).to have_content '12345'
+      expect(page).to have_content 'Visitante'
+      expect(page).to have_content 'Torre A - 12'
+      expect(page).to have_content 'Maria Silveira'
+      expect(page).to have_content I18n.l(1.day.from_now.to_date)
+    end
+    expect(page).not_to have_css "#visitor-#{fourth_visitor.id}"
+    expect(page).not_to have_content 'Juliana Ferreira'
+  end
+
+  it 'and sees if list is empty' do
```

Comment:
Ficou meio estranho a junção desse `it` com o `describe`, aqui poderia ser algo como: `'even if the list is empty'` ou `'and see a message if the list is empty'`.